src/hooks/useModal.tsx

```diff
@@ -0,0 +1,8 @@
+import { useState } from 'react';
+
+const useModal = () => {
+  const [isOpenModal, setIsOpenModal] = useState(true);
```

Comment:
```suggestion
const useModal = (initValue = true) => {
  const [isOpenModal, setIsOpenModal] = useState(initValue);
```

`isOpenModal` 상태의 초기 상태를 매개변수로 받을 수 있게 확장을 열어두는게 좋지 않을까요?